compiler/src/test/scala/org/scalajs/nscplugin/test/DiverseErrorsTest.scala

```diff
@@ -26,13 +26,8 @@ class DiverseErrorsTest extends DirectTest with TestHelpers  {
 
   private def version = scala.util.Properties.versionNumberString
 
-  private val allowsSingletonClassOf = (
-      !version.startsWith("2.12.") &&
-      version != "2.13.0" &&
-      version != "2.13.1" &&
-      version != "2.13.2" &&
-      version != "2.13.3"
-  )
+  private val allowsSingletonClassOf =
+    !version.startsWith("2.12.") && version != "2.13.3"
```

Comment:
This is the only occurrence of a special-case for 2.13.3. Since we need the code paths for 2.12.x anyway, IMO there's no reason to drop 2.13.3.